apps/aecore/src/aec_peer_connection_listener.erl

```diff
@@ -0,0 +1,51 @@
+%%%=============================================================================
+%%% @copyright 2018, Aeternity Anstalt
+%%% @doc
+%%%    Module holding the entry point for incoming P2P connections
+%%% @end
+%%%=============================================================================
+-module(aec_peer_connection_listener).
+
+-export([start_link/0, stop/0]).
+
+start_link() ->
+    Pid = spawn_link(fun pc_listener/0),
+    register(?MODULE, Pid),
+    {ok, Pid}.
+
+stop() ->
+    ?MODULE ! stop,
+    ok.
+
+pc_listener() ->
+    {_Scheme, Host, _Port} = aeu_env:local_peer(),
+    {ok, SecKey} = aec_keys:peer_privkey(),
+    {ok, PubKey} = aec_keys:peer_pubkey(),
+    Port = aec_peers:sync_port(),
+    lager:info("Starting peer_connection_listener at port ~p", [Port]),
+    {ok, LSock} = gen_tcp:listen(Port, [{active, false}, binary, {reuseaddr, true}]),
```

Comment:
Configurable max number of acceptors?
Configurable max timeout(s)?
Using ranch instead of gen_tcp for having only one code path for connections?

Reply:
This will be addressed further on in https://www.pivotaltracker.com/story/show/156222652

Reply:
I would also like to see configurable `listen_address`, like all other listen sockets.

From ops perspective it's much easier to set the `listen_address` than configuring firewall rules. (e.g. in case of private setups)